src/main/java/io/aiven/kafka/connect/s3/config/S3SinkConfig.java

```diff
@@ -99,6 +99,7 @@ public class S3SinkConfig extends AivenCommonConfig {
     public static final String AWS_ACCESS_KEY_ID_CONFIG = "aws.access.key.id";
     public static final String AWS_SECRET_ACCESS_KEY_CONFIG = "aws.secret.access.key";
     public static final String AWS_S3_BUCKET_NAME_CONFIG = "aws.s3.bucket.name";
+    public static final String AWS_S3_SSEA_NAME_CONFIG = "aws.s3.ssea.name";
```

Comment:
Let's keep it aligned with the AWS SDK API
```suggestion
    public static final String AWS_S3_SSE_ALGORITHM_CONFIG = "aws.s3.sse.algorithm";
```
or similar. Eventually, we may need to add customer-provided keys, and piggyback on `sse` prefix .